src/components/CesiumViewer.vue

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+import { defineComponent, ref, reactive } from '@vue/composition-api';
+
+export default defineComponent({
+  name: 'CesiumViewer',
+
+  setup() {
+    const animation = ref(true);
+    const timeline = ref(true);
+    const camera = reactive({
+      position: {
+        lng: -122.4175,
+        lat: 37.655,
+        height: 100000,
+      },
+      heading: 360,
+      pitch: -90,
+      roll: 0,
+    });
+
+    const ready = (cesiumInstance: { Cesium: any; viewer: any }) => {
```

Comment:
Ideally we'd properly type `Cesium` and `viewer`. However since it seems that's impractical, we should at least ignore the eslint warnings for using `any` 
```suggestion
    // eslint-disable-next-line @typescript-eslint/no-explicit-any
    const ready = (cesiumInstance: { Cesium: any; viewer: any }) => {
```

Reply:
@marySalvi Are type definitions actually available for Cesium? https://cesium.com/blog/2020/06/01/cesiumjs-tsd/